fuzz-testing/src/main/scala/org/apache/comet/fuzz/Meta.scala

```diff
@@ -137,6 +137,10 @@ object Meta {
             SparkBinaryType,
             SparkArrayType(
               SparkTypeOneOf(Seq(SparkStringType, SparkNumericType, SparkBinaryType))))))),
+    createFunctionWithInputTypes(
+      "concat",
+      Seq(SparkStringType, SparkStringType)
+    ), // TODO: variadic
```

Comment:
I know that this PR is just to support string inputs in Comet concat, but the fuzz tester should ideally test for all types that Spark supports